### ISSUE TITLE
[Bug Report][3.5.4] v-date-picker buttons in v-date-picker-controls doesn't have aria-label

### ISSUE THREAD

**Issue by maja-fredstrom** [vuetifyjs/vuetify#19251]
### Environment
**Vuetify Version:** 3.5.4
**Vue Version:** 3.4.19
**Browsers:** Chrome 121.0.0.0
**OS:** Windows 10

### Steps to reproduce
Create a simple date picker. 
inspect the code. 

### Expected Behavior
Button for picking year 
and buttons for navigating between months should be accessible.

### Actual Behavior
Buttons doesn't have discernible text

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNpVj8EKwjAQRH9l3UsVbItaD0oVvPkHHqyH0m4lmDQhWYsg/rtJq9DedobHzOz1jc5W6cmYpHsS7jFnUkaWTMeiBci7uDSmP3tR6ZZL0ZL9Wb3J9OK4ESRr6GKla5KHApW7FwjpmKt9amxE9SALJKkrWejWo+uswGOeToB/pXenncEZJuXpaKqXrrLCMDji57BYKKMtwxssNfCBxmoFkf8yCjiAz3UMficcAjGPziSlhou2sp5Fi1AwRPow/Cxxk2TJaofh2CYZ3r5TBmWK)
